admin_guide/cluster_management.rst

```diff
@@ -575,6 +575,43 @@ In addition to our core Citus extension, we also maintain several others:
 * `postgresql-topn <https://github.com/citusdata/postgresql-topn>`_ - Returns the top values in a database according to some criteria. Uses an approximation algorithm to provide fast results with modest compute and memory resources.
 * `postgresql-hll <https://github.com/citusdata/postgresql-hll>`_ - HyperLogLog data structure as a native data type. It's a fixed-size, set-like structure used for distinct value counting with tunable precision.
 
+.. _create_db:
+
+Creating a New Database
+=======================
+
+Each PostgreSQL server can hold multiple databases. Remember that new databases do not inherit the extensions of any others; all desired extensions must be added afresh. To run Citus on a new database, you'll need to create the database on the coordinator and workers, create the Citus extension within that database, and register the workers in the coordinator database.
```

Comment:
Add a link to PG docs maybe? The concept of a database might be new.
https://www.postgresql.org/docs/current/static/manage-ag-overview.html